src/Components/Aspire.Azure.Storage.Blobs/AspireBlobStorageExtensions.cs

```diff
@@ -12,6 +13,15 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Diagnostics.HealthChecks;
 
+[assembly: ConfigurationSchema(
```

Comment:
NIT: This is fine too, but we could also have files called AssemblyInfo.cs for these lines like we do in other repos.

Reply:
Moved to AssemblyInfo.cs files.